sys-apps/irqbalance/files/irqbalance.service

```diff
@@ -2,10 +2,8 @@
 Description=CPU Interrupt Balancer
 
 [Service]
-ExecStart=/usr/sbin/irqbalance --debug
-# --debug implies a lot of cruft
-# but it's the only way to get foregrounding
-StandardOutput=null
+EnvironmentFile=-/etc/irqbalance.env
+ExecStart=/usr/sbin/irqbalance --foreground $IRQBALANCE_ARGS
```

Comment:
Yes, I think the unit file predates introduction of `--foreground`. In fact, it seems that I submitted the patch adding it even.